application/templates/searchdeed.html

```diff
@@ -1,14 +1,14 @@
 {% extends "base/lr_base_layout.html" %}
 
 {% block content %}
-<div class="main" id="content">
+<main id="content" role="main">
```

Comment:
suggested indentation 

```
{% extends "base/lr_base_layout.html" %}
{% block content %}
<main id="content" role="main">

	{{ super() }}
	<div>&nbsp;</div>
	
	{% if error %}
	<div class="text error-summary" role="group" aria-labelledby="error-deed-not-found" tabindex="-1">
		<h1 class="heading-medium error-summary-heading" id="error-deed-not-found">Unable to find your mortgage deed</h1>
		<p>This might be because:</p>
		<ul class="list list-bullet">
			<li>the borrower reference was mistyped</li>
			<li>or the date of birth was not in the format dd mm yyyy</li>
		</ul>
		<p>
			If you continue to experience problems please contact your
			conveyancer.
		</p>
	</div>
	{% endif %}

	<form id="search-deed-form" action="/date-of-birth" method="POST">
		<label class="form-label-bold" for="borrower_token">
			What is your borrower reference?
		</label>
		<span class="form-hint">
			For example, 2pb47w
		</span>
		<div class="form-group">
			<input id="borrower_token" class="form-control" type="text" name="borrower_token">
		</div>
		<div class="form-group">
			<input type="submit" class="button" value="Continue">
		</div>
	</form>

	<details>
		<summary>
			I don't have a borrower reference
		</summary>
		<p>If you don't have a reference your should contact your conveyancer.</p>
	</details>

</main>
{% endblock %}
```